explorer/nodewatch/nodewatch/NetworkConnector.py

```diff
@@ -62,7 +65,7 @@ async def _update_height(self, session, descriptor):
 
 				if 'finalized_height' in response_json:
 					descriptor.finalized_height = int(response_json['finalized_height'])
-		except (aiohttp.client_exceptions.ClientConnectorError, asyncio.TimeoutError) as ex:
+		except (aiohttp.client_exceptions.ClientConnectorError, aiohttp.client_exceptions.ContentTypeError, asyncio.TimeoutError) as ex:
```

Comment:
why there's ContentType exception?

Reply:
was hitting it yesterday when nodes were stopped/updating but webserver was still running and returning text/html 502 page 😬